migrate/live-migration/live-migration-from-postgres.md

```diff
@@ -12,402 +12,158 @@ import StepOne from "versionContent/_partials/_migrate_dual_write_step1.mdx";
-or `DELETE` operations on the table we **do not recommend** using `FULL`. For
-each `UPDATE` or `DELETE` statement, Postgres will have to read the whole table
-to find all matching rows, which will result in significantly slower
-replication.
-
-```sql
-ALTER TABLE {table_name} REPLICA IDENTITY FULL
-```
-
-Once you're sure all your tables are properly configured for replication, use
-`pgcopydb`'s follow command to create a replication slot:
-
-```sh
-pgcopydb follow \
-  --source "$SOURCE" \
-  --target "$TARGET" \
-  --fail-fast \
-  --plugin wal2json
-```
-
-<SetupSourceTarget />
-
-This command is going to be active during most of the migration process. You
-can run it on a separate terminal instance, or start it in the background. To
-start it in the background append `> pgcopydb_follow.log 2>&1 &` to the
-command. For example:
-
-```sh
-pgcopydb follow \
-  --source "$SOURCE" \
-  --target "$TARGET" \
-  --fail-fast \
-  --plugin wal2json > pgcopydb_follow.log 2>&1 &
+Execute the following command for each table you want to convert on the target DB:
+    `SELECT create_hypertable('<table name>', '<time column name>', chunk_time_interval => INTERVAL '<chunk interval>')`
 ```
 
-The `> pgcopydb_follow.log 2>&1` part redirects all the messages to the
-`pgcopydb_follow.log` file, this is optional but recommended. The `pgcopydb
-follow` command outputs many messages, if they are not redirected, using the
-terminal becomes cumbersome due to the constant pop up of messages.
-
-The `follow` command not only creates the replication slot for streaming
-changes, but also exports a [snapshot][snapshot] ID to `/tmp/pgcopydb/snapshot`.
-This ID can be used to migrate the data that was stored in the database prior
-to the creation of the replication slot.
-
-> A snapshot determines which data is visible to the transaction that is using
-the snapshot. Synchronized snapshots are necessary when two or more sessions
-need to see identical content in the database.
-
-Before the stream of changes can be applied, the schema and data that existed
-prior to the creation of the replication slot must be migrated ([step
-4][step-4]). The point that marks the beginning of the replication and
-buffering of changes is given by the exported snapshot. The larger the
-database, the more time it takes to perform the initial migration, and the
-longer the buffered files need to be stored.
-
-## 4. Migrate roles and schema from source to target
+Ideally, tables that contain time-series data should be converted to Hypertables.
+You need to run `create_hypertable()` for each table that you want to convert to
+a Hypertable in the target database. For more information, see [Hypertable docs].
 
-### 4a. Dump the database roles from the source database
+Once you have finished creating Hypertables, you need to signal "continue" to proceed.
+You can do it either by pressing `c` key or giving a `SIGUSR1` signal,
```

Comment:
I find it confusing that we provide both options "pressing `c` or sending `SIGUSR1`. Why do we have both options? Can we just skip mentioning `SIGUSR1`?

Reply:
+1, since we advocate to use docker in interactive mode(-it), there is no need to mention about signals.